dash/dash.py

```diff
@@ -711,10 +711,19 @@ def serve_component_suites(self, package_name, path_in_package_dist):
             package.__path__,
         )
 
-        return flask.Response(
+        response = flask.Response(
             pkgutil.get_data(package_name, path_in_package_dist),
             mimetype=mimetype,
         )
+        response.add_etag()
+        (tag,) = response.get_etag()[:1]
```

Comment:
not sure what the preferred syntax is for unpacking only part of a tuple

Reply:
```suggestion
        tag = response.get_etag()[0]
```

Reply:
we should avoid the unpacking here as it's a little bit tricky to handle the syntax in python 2 and 3. with pure python3 it's easier.

Reply:
https://github.com/plotly/dash/pull/973/commits/5e61384996035ff47542134cf5e2a8ad978c1a5c